sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationAudience.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Data.AppConfiguration
+{
+    /// <summary> Cloud audiences available for AppConfiguration. </summary>
+    public readonly partial struct ConfigurationAudience : IEquatable<ConfigurationAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConfigurationAudience"/> object.
+        /// </summary>
+        /// <param name="value">The Azure Active Directory audience to use when forming authorization scopes.
+        /// For the App Configuration service, this value corresponds to a URL that identifies the Azure cloud where the resource is located.</param>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        /// <remarks>Please use one of the constant members over creating a custom value unless you have special needs for doing so.</remarks>
+        public ConfigurationAudience(string value)
```

Comment:
nittiest of nits:  We generally put const and static properties above instance members.   I don't have a strong opinion, but if it were me, I'd bump the constructor down below L38.    That said, I leave it to your discretion.